action.yml

```diff
@@ -83,15 +91,59 @@ outputs:
 runs:
   using: composite
   steps:
+  - name: Log the next step action
+    run: echo ▷ Figuring out the environment Python version...
+    shell: bash
+  - name: Compute job Python version
+    id: compute-job-python-version
+    run: |
+      # Compute job Python version
+      import sys
+
+      # Input from GHA
+      job_python_version = '${{ inputs.job-python-version }}'
+      old_python_version = '${{ inputs.python-version }}'
+      ansible_core_version = '${{ inputs.ansible-core-version }}'
+
+      # Handle case that deprecated Python version parameter is provided
+      if old_python_version:
+          # Check whether the default of job-python-version was changed
```

Comment:
This code branch needs a warning issued via a workflow command. See https://github.com/ansible-community/ansible-test-gh-action/pull/19#discussion_r941453825 / https://github.com/pypa/gh-action-pypi-publish/commit/9b8e7336db3f96a2939a3e9fa827c62f466ca60d / https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-a-warning-message.

Reply:
Why? https://github.com/ansible-community/ansible-test-gh-action/pull/19/files#diff-1243c5424efaaa19bd8e813c5e6f6da46316e63761421b3e5f5c8ced9a36e6b6R56 ensures that a warning is shown once that option is used. See https://github.com/ansible-community/ansible-test-gh-action/actions/runs/2851274141 for how that looks like.

Reply:
Oh, I must've missed that. Didn't know that `deprecationMessage` was a thing. TIL. In that case, you're right. No hacks like that are necessary.